src/components/Container.js

```diff
@@ -147,22 +140,22 @@ function Container(_props) {
   );
 }
 
-Container.propTypes = {
-  boxShadow: PropTypes.oneOf(BOX_SHADOWS),
-  ...responsiveMarginType,
-  ...responsivePaddingType,
-  ...responsiveWidthType,
-  ...responsiveHeightType,
-  ...responsivePropType("bg", PropTypes.oneOf(BACKGROUNDS)),
-  ...responsivePropType("flexGrow", PropTypes.bool),
-  ...responsivePropType("hasBorder", PropTypes.bool),
-  ...responsivePropType("textStyle", PropTypes.oneOf(TEXT_STYLES)),
-  ...responsivePropType("textAlign", PropTypes.oneOf(TEXT_ALIGNS)),
-  ...responsivePropType("overflow", PropTypes.string),
-  ...responsivePropType("hasBreakpointWidth", PropTypes.bool),
-  ...responsivePropType("hide", PropTypes.bool),
-  children: PropTypes.node,
-  testId: PropTypes.string,
-};
+// Container.propTypes = {
+//   boxShadow: PropTypes.oneOf(BOX_SHADOWS),
+//   ...responsiveMarginType,
+//   ...responsivePaddingType,
+//   ...responsiveWidthType,
+//   ...responsiveHeightType,
+//   // ...responsivePropType("bg", PropTypes.oneOf(BACKGROUNDS)),
+//   ...responsivePropType("flexGrow", PropTypes.bool),
+//   ...responsivePropType("hasBorder", PropTypes.bool),
+//   ...responsivePropType("textStyle", PropTypes.oneOf(TEXT_STYLES)),
+//   ...responsivePropType("textAlign", PropTypes.oneOf(TEXT_ALIGNS)),
+//   ...responsivePropType("overflow", PropTypes.string),
+//   ...responsivePropType("hasBreakpointWidth", PropTypes.bool),
+//   ...responsivePropType("hide", PropTypes.bool),
+//   children: PropTypes.node,
+//   testId: PropTypes.string,
+// };
```

Comment:
Why did you remove all the PropTypes here? Wondering what's special about this `Container` component 🤔

Reply:
Container is referenced in in `src/components/Text.test.tsx` so we have to have this change to unblock this ticket.